composer.json

```diff
@@ -28,14 +28,17 @@
         "opentracing/opentracing": "1.0.0-beta5",
         "packaged/thrift": "^0.10",
         "phlib/base_convert": "^1.0",
+        "psr/cache": "^1.0",
```

Comment:
Thoughts on adding `"ext-hrtime": "*"` here?
Given that its only needed for < PHP 7.3, this might not be necessary

Reply:
I tought about it, but `hrtime` is required only if you want to use RateLimiter. Also it is not required if you use php 7.3 :). I added it to README.md in RateLimiter section.
If you think, we should add it to composer.json, we can, I have no problem with it.